feature/repair/src/main/java/com/iteneum/repair/presentation/RepairView.kt

```diff
@@ -7,35 +7,68 @@ import androidx.compose.runtime.*
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.res.stringArrayResource
 import androidx.compose.ui.res.stringResource
-import androidx.core.text.isDigitsOnly
+import androidx.hilt.navigation.compose.hiltViewModel
+import com.iteneum.RepairRequest
+import com.iteneum.UIEventRepair
 import com.iteneum.designsystem.components.*
 import com.iteneum.designsystem.theme.LPTypography
 import com.iteneum.designsystem.theme.LeasePertTheme
 import com.iteneum.repair.R
+import com.iteneum.repair.data.RepairViewModel
 
 /**
  * Function that creates [RepairView] compose. for user's service request.
  * User will require to fill all the correspondent fields on the screen.
  *
+ * Function [RepairView] has
+ * @param navigateToApartment - to get the function to navigate ApartmentView when clicking button
+ * @param repairViewModel - to get the Repair View Model access
+ *
+ * Function [RepairContainer] has
```

Comment:
Move RepairContainer documentation to their respective function.

Reply:
Ok, moved bellow